plugins/destination/postgresql/client/migrate.go

```diff
@@ -167,6 +170,50 @@ func (c *Client) dropTable(ctx context.Context, tableName string) error {
 	return nil
 }
 
+func (c *Client) migrateToCQID(ctx context.Context, tableName string, _ schema.Column) error {
+	// Steps:
+	// acquire connection
+	conn, err := c.conn.Acquire(ctx)
+	if err != nil {
+		return fmt.Errorf("failed to acquire connection: %w", err)
+	}
+	defer conn.Release()
+	// start transaction
+	tx, err := conn.BeginTx(ctx, pgx.TxOptions{})
```

Comment:
maybe use serializable explicitly rather than relying on the default level being enough?